### PR TITLE
[#253] Removed extra </link> tag from index.html

### PR DESCRIPTION
**Pull request by prajeshElEvEn**
Fix for [Unable to format index.html #253](https://github.com/exo-explore/exo/issues/253) issue.

Removed extra closing `</link>` tag from `index.html`, thus enabling the file to be formatted by prettier.

### CODE REVIEW

File: tinychat/examples/tinychat/index.html [hunk 1]
```diff
@@ -1,111 +1,168 @@
...
-<script src="/index.js"></script>
-<link href="/static/fonts.googleapis.com" rel="preconnect"/>
-<link crossorigin="" href="/static/fonts.gstatic.com" rel="preconnect"/>
-<link href="/static/fonts.googleapis.com/css2" rel="stylesheet"/>
-<link href="/static/cdn.jsdelivr.net/npm/purecss@3.0.0/build/base-min.css" rel="stylesheet"/>
-<link crossorigin="anonymous" href="/static/cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" referrerpolicy="no-referrer" rel="stylesheet">
-<link href="/static/unpkg.com/@highlightjs/cdn-assets@11.9.0/styles/vs2015.min.css" rel="stylesheet"/>
-<link href="/index.css" rel="stylesheet"/>
-<link href="/common.css" rel="stylesheet"/>
-</link></head>
```
Comment (larson-carter) [inline tinychat/examples/tinychat/index.html:25]: Does prettier not want the closing link and head tag on separate lines?

File: tinychat/examples/tinychat/index.html [hunk 2]
```diff
@@ -1,111 +1,168 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (larson-carter) [inline tinychat/examples/tinychat/index.html:1]: Why is prettier modifying this as well?

File: tinychat/examples/tinychat/index.html (hunk 1 above)
Comment (prajeshElEvEn) [reply tinychat/examples/tinychat/index.html:25]: The `link` tag is self closing and does not require a closing tag. Hence, there is no need for the `<\link> tag.

Even if you put them on separate lines, the issue of not formatting the code still persists.

So only thing we can do it to get rid of it.

File: tinychat/examples/tinychat/index.html (hunk 2 above)
Comment (prajeshElEvEn) [reply tinychat/examples/tinychat/index.html:1]: That is a default behavior of prettier, you can take a look a this [issue](https://github.com/prettier/prettier/issues/15096) on prettier's github.

If you wish I can change the declaration back to uppercase, however, anytime a person formats it via prettier, it is going to be in lowercase again.

Hence, a permanent solution would be to add `<!-- prettier-ignore -->` above the `<!DOCTYPE html>` declaration.

**Comment by AlexCheema** [comment]
Thanks for this!
This should be a one line change.
Please fix conflicts and keep formatting the same as it was.

**Comment by prajeshElEvEn** [comment]
> Thanks for this! This should be a one line change. Please fix conflicts and keep formatting the same as it was.

Hey @AlexCheema, I have reset the formatting and removed only the closing `</link>` tag.
Let me know, if this works.

**Comment by AlexCheema** [comment]
lgtm after conflicts resolved.

**Pull request closed without merging**